perspective_reddit_bot/check_mod_actions.py

```diff
@@ -30,10 +30,23 @@
 import moderate_subreddit
 
 
-
+# approved, removed, and deleted columns are booleans. 'approved' is only
+# present when the bot is a mod.
 APPROVED_COL = 'approved'
 REMOVED_COL = 'removed'
 DELETED_COL = 'deleted'
+# score, ups, and downs refer to user votes. in practice, it appears that
+# 'downs' is always 0 and 'score' == 'ups'. (this may be different if the bot is
+# a mod.)
+SCORE_COL = 'score'
+UPS_COL = 'ups'
+DOWNS_COL = 'downs'
+SCORE_HIDDEN_COL = 'score_hidden'
```

Comment:
Add a description of score_hidden? I assume it's a boolean indicating if the score is hidden during the initial stages of a comment's life? 
https://www.reddit.com/r/redditdev/comments/1dfdpp/api_change_score_hidden_added_to_comments_and/

Reply:
done